config/iam/recommended-inline-policy

```diff
@@ -0,0 +1,10 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "ec2:ModifyNetworkInterfaceAttribute",
+      "Resource": "*"
+    }
+  ]
+}
```

Comment:
I believe we should include this along with all the policies listed in https://github.com/aws-controllers-k8s/mq-controller/blob/main/config/iam/recommended-policy-arn , and get rid the `recommended-policy-arn`

Reply:
Hi @A-Hilaly take into account that the one `arn:aws:iam::aws:policy/AmazonMQFullAccess` present in https://github.com/aws-controllers-k8s/mq-controller/blob/main/config/iam/recommended-policy-arn is not an inline policy, I thought this was the reason we have the two